test/preprocessing/data.test.ts

```diff
@@ -7,7 +7,7 @@ import {
   normalize,
   OneHotEncoder,
   PolynomialFeatures,
-} from '../../src/lib/preprocessing';
+} from '../../src/lib/preprocessing/index';
```

Comment:
'../../src/lib/preprocessing' and '../../src/lib/preprocessing/index' are the same thing, except the first one is more readable

Reply:
/preprocessing is the link to the directory whereas /index links it to the appropriate typescript file.

Reply:
I was getting an error because it couldn’t find the appropriate file so I had to change it to /index

Reply:
> /preprocessing is the link to the directory whereas /index links it to the appropriate typescript file.

No, It's not. 
Consult this link https://www.typescriptlang.org/docs/handbook/module-resolution.html for reference.